lib/configure.js

```diff
@@ -272,6 +272,7 @@ function configure (gyp, argv, callback) {
       argv.push('-Dnode_gyp_dir=' + nodeGypDir)
       argv.push('-Dnode_lib_file=' + release.name + '.lib')
       argv.push('-Dmodule_root_dir=' + process.cwd())
+      argv.push('-Dnode_engine=' + (gyp.opts.node_engine || process.jsEngine || 'v8'))
```

Comment:
Okay, I think I follow.  Still, using `process.jsEngine` here means we potentially build the add-on for the wrong binary, right?  Not the end of the world but I need to know if my understanding is correct.